src/Components/SmartComponents/Modals/BaseModal.js

```diff
@@ -37,6 +37,7 @@ export function useJustificationInput(initialValue) {
                 value={justification}
                 resizeOrientation="vertical"
                 aria-label={'justification note'}
+                style={{ minHeight: '2.25rem' }}
```

Comment:
did you add the inline style due to how chrome mounts modals?

Reply:
@katerinapat yes, exactly